src/app/member/[memberIdentifier]/study-request/yoda.tsx

```diff
@@ -0,0 +1,42 @@
+'use client'
+
+import { Text } from '@mantine/core'
+import { useEffect } from 'react'
+import { notifications } from '@mantine/notifications'
+
+// use a constant object to store the state of the component
+// this prevents re-displaying if they navigate away and back
+const state = {
+    shown: false,
+}
+
+export const YodaNotice = () => {
+    useEffect(() => {
+        if (state.shown) return
+        state.shown = true
```

Comment:
yoda.tsx could maybe use [useState](https://react.dev/reference/react/useState) hook?!

Reply:
I actually did on a first version, but then it re-displays if you navigate away and come back, IMO it should only show once